Sync/Synchronizers/Bookmarks/BookmarksSynchronizer.swift

```diff
@@ -59,23 +59,22 @@ class TrivialBookmarkStorer: BookmarkStorer {
         var success: [GUID] = []
         var failed: [GUID: String] = [:]
 
-        func onUpload(result: POSTResult) -> DeferredTimestamp {
-            modified = result.modified
+        func onUpload(result: POSTResult, lastModified: Timestamp?) -> DeferredTimestamp {
             success.appendContentsOf(result.success)
             result.failed.forEach { guid, message in
                 failed[guid] = message
             }
 
-            log.debug("Uploaded records got timestamp \(modified).")
-            local.setModifiedTime(modified, guids: result.success)
-            return deferMaybe(result.modified)
+            log.debug("Uploaded records got timestamp \(lastModified).")
+            local.setModifiedTime(lastModified ?? modified, guids: result.success)
```

Comment:
Nope. I guess the annoying thing was that `modified` was non-optional before but now because its coming down as a header value its optional. What are your thoughts on force unwrapping the XLM timestamp? Doing this would match the previous behavior of always having a timestamp but introduce the unfortunate effect of a crash if it's not there. There isn't really an empty state for a timestamp.